bigquery_etl/docs/generate_docs.py

```diff
@@ -98,7 +98,7 @@ def main():
                         # project level-doc file
                         project_doc_dir = out_dir / path / name
                         project_doc_dir.mkdir(parents=True, exist_ok=True)
-                        dest = project_doc_dir / "overview.md"
+                        dest = project_doc_dir / "about.md"
```

Comment:
By changing the filename from "overview" to "about", the description for mozfun project jumps to the top before other function description, which makes more sense imo